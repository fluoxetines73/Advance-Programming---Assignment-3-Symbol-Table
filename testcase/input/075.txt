INSERT y number
ASSIGN y 123 random
BEGIN
PRINT
ASSIGN a unimportant
END
ASSIGN y 987654
BEGIN
LOOKUP y
PRINT
RPRINT
INSERT a number