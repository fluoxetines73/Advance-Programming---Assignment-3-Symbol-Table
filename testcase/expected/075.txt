Invalid: ASSIGN y 123 random